task19/js/module/error.js

```diff
@@ -0,0 +1,4 @@
+export const renderErrorMessage = (parent, errorMessage) => {
```

Comment:
I think 'modules' is a more appropriate directory name.

Reply:
Correction. It should be written in the plural.
https://github.com/seeeen044/JavaScript_handson/commit/b4d7d86b721fa44804f102498636163c7181f778